include/rcutils/time.h

```diff
@@ -100,6 +100,84 @@ RCUTILS_WARN_UNUSED
 rcutils_ret_t
 rcutils_steady_time_now(rcutils_time_point_value_t * now);
 
+/// Return a time point as nanoseconds in a string.
+/**
+ * The number is always fixed width, with left padding zeros up to the maximum
+ * number of digits the time point can represent.
+ * Right now that is 19 digits (so 19 characters) for a signed 64-bit integer.
+ * Negative values will have a leading `-`, so they will be one character
+ * longer than the positive values.
+ *
+ * The recommended minimum size of the input string is 32 characters, but
```

Comment:
How is this "recommended minimum size" relevant in the docblock? Isn't is sufficient to document the exact max size necessary? Also why 32?

Same below.

Reply:
Because the usable block size on my machine is `16` bytes, and `32` is the smallest multiple of that in which the exact size of this string fits. Also, it gives us a little flexibility for the future, e.g. if we wanted to add `+` to the front of the time to distinguish it from `-`. Not saying we will, but if when you ask for 21 characters malloc gives you 32 anyways...

This is the program I used to check:

```c++
#include <cassert>
#include <cstdio>

#if __linux__
#include <malloc.h>
#else
#include <malloc/malloc.h>
#endif

int main()
{
  uint8_t * bytes = (uint8_t *)malloc(1);
  assert(bytes);
#if __linux__
  printf("%zu\n", malloc_usable_size(bytes));
#else
  printf("%zu\n", malloc_size(bytes));
#endif
  return 0;
}
```

I didn't try it on Windows.